### ISSUE TITLE
restrict_to_dates isn't restrictive enough

### ISSUE THREAD

**Issue by shakedk** [mrcagney/gtfs_kit#6]
Firstly, great module! I use it a lot and it's wonderful :)

I'm trying to restrict a gtfs feed using miscellany.restrict_to_dates with a single date.
In the feed I'm using, there's calendar dates file that looks like this
44811,20200413,1
43907,20200413,1
45873,20200413,1
43889,20200329,1
44009,20200329,1
44046,20200329,1
44060,20200329,1
44899,20200329,1
46342,20200329,1
44454,20200412,1
43907,20200412,1
43977,20200412,1
....

I tried to restrict the feed to 20200412 only, but I it generated a feed with all the other days the services on this days operate in. For example, service id 43907 runs on both 20200412 and 20200413.
Is there any way to create a feed that is filtered only for the specific day I'm providing the function with? So for my example only the last 3 rows will be present in the outocme

**Comment by araichev** [comment]
Hi @shakedk, thanks for your constructive feedback.
I think i just need to change to this block of code in the function ``restrict_to_dates()``:

```
    if feed.calendar_dates is not None:
        feed.calendar_dates = feed.calendar_dates.loc[
            lambda x: x.service_id.isin(service_ids) & x.date.isin(dates)
        ].copy()
```

I'll do this in the next few days.

**Comment by araichev** [comment]
Done and published in release 5.0.0.
